frontend/beCompliant/src/components/questionPage/QuestionDetails.tsx

```diff
@@ -0,0 +1,26 @@
+import { Text, Stack } from '@kvib/react';
+import { formatDateTime } from '../../utils/formatTime';
+import { Question } from '../../api/types';
+
+type Props = {
+  question: Question;
+  answerUpdated: string;
+};
+
+export function QuestionDetails({ question, answerUpdated }: Props) {
+  const sikkerhetskontroller = question.metadata.optionalFields?.find(
+    (field) => field.key == 'Sikkerhetskontroller'
+  )?.value[0];
+
+  return (
+    <Stack marginBottom="120px" gap="4px">
```

Comment:
```suggestion
    <Stack marginBottom="30" gap="1">
```

Reply:
hvorfor er det sån stor marginBottom?

Reply:
Jeg hade flyttet denne marginen ut til parent containern, denne componenten kanske ikke skal ha 120px margin alltid. Jeg hade brukt  Props  extends StackProps og siden hatt `...rest`
```suggestion
    <Stack gap="4px" {...rest}>
```